CONTRIBUTING.md

```diff
@@ -92,7 +92,7 @@ We make the following distinction for code that exists in our repository:
 Code all lives in the [dotnet/samples](https://github.com/dotnet/samples) repository. We are working toward a model where our samples folder structure matches our docs folder structure. Standards that we follow are:
 
 - The top level *snippets* folder contains snippets for small, focused samples.
-- API reference samples have been in a folder following this pattern: *snippets/<language>/api/<namespace>/<apiname>*.
+- API reference samples have been in a folder following this pattern: *snippets/\<language>/api/\<namespace>/<apiname>*.
```

Comment:
@svick one more to escape for `<apiname>`

Reply:
@pkulikov Thanks, I should have noticed that myself. Fixed.